docs/.vitepress/config.ts

```diff
@@ -0,0 +1,168 @@
+/**
+ * Copyright 2023 Mineiros GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { HeadConfig } from 'vitepress'
+import { defineConfig } from 'vitepress'
+
+function getPath(path: string) {
+  const uri = path.replace(/(?:(^|\/)index)?\.md$/, '$1')
+
+  return uri === 'index' ? '' : uri
+}
+
+// https://vitepress.dev/reference/site-config
+export default defineConfig({
+  title: 'Terramate',
+  description: 'A VitePress Site',
```

Comment:
```suggestion
  description: 'The Terramate Documentation',
```
It's [overridden](https://vitepress.dev/reference/site-config#description) by frontmatter but having a default makes sense in case it's not defined in some specific markdown.

Reply:
makes sense will change in a follow up PR